packages/contracts/test/helpers/LiveTestHelper.sol

```diff
@@ -25,10 +25,11 @@ import "../../src/dollar/ExcessDollarsDistributor.sol";
 import "../../src/dollar/SushiSwapPool.sol";
 import "../../src/dollar/interfaces/IERC1155Ubiquity.sol";
 
-import "Uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
-import "Uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
+import
+    "../../lib/Uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
+import "../../lib/Uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
```

Comment:
Lets revert these changes. As I said above, `import` is getting handled described in remapping.txt. we can't update for just spell check outside of origin concept of Fuondry